rules/retrieve.smk

```diff
@@ -728,6 +728,77 @@ if config["enable"]["retrieve"]:
         run:
             move(input[0], output[0])
 
+    rule seawater_temperature:
+        output:
+            seawater_temperature="data/seawater_temperature.nc",
+        log:
+            "logs/retrieve_seawater_data.log",
+        resources:
+            mem_mb=10000,
+        retries: 2
+        shell:
+            """
+            wget -nv -c https://zenodo.org/records/15198744/files/seawater_temperature.nc -O {output.seawater_temperature}
+            """
```

Comment:
Can't use `wget` as we can't expect it to be available on Windows. Rewrite like:

```
    rule retrieve_synthetic_electricity_demand:
        input:
            storage(
                "https://zenodo.org/records/10820928/files/demand_hourly.csv",
            ),
        output:
            "data/load_synthetic_raw.csv",
        log:
            "logs/retrieve_synthetic_electricity_demand.log",
        resources:
            mem_mb=5000,
        retries: 2
        run:
            move(input[0], output[0])
```